### ISSUE TITLE
Voiceover: VAC logo in top banner is not accessible

### ISSUE THREAD

**Issue by brdunfield** [veteransaffairscanada/vac-benefits-directory#1634]
No alt-text or indication that it is a link when landing on the page

**Comment by SupeDeDupe** [comment]
Is it supposed to be a link?

<img width="428" alt="screen shot 2018-12-21 at 10 04 31 am" src="https://user-images.githubusercontent.com/17673146/50346157-dc461080-0507-11e9-8a78-884c505c067b.png">

**Comment by SupeDeDupe** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)

**Comment by brdunfield** [comment]
This image is no longer a link - this can potentially be closed?

**Comment by courtneycl** [comment]
closing - as bethany said, no longer a link, and we'll be swapping the header anyway.
